autospec/specfiles.py

```diff
@@ -394,15 +390,18 @@ def write_prep(self, ruby_pattern=False):
                 self._write_strip("%setup -q -c -n " + self.tarball_prefix)
             else:
                 self._write_strip("%setup -q -n " + self.tarball_prefix)
+                for archive in self.sources["archive"]:
+                    self._write_strip("%setup -D -T -n {0} -a {1}"
```

Comment:
Please pass the `-q` option too, so that the tarball/zip extraction does not spam the build log.